### PR TITLE
feat: merge method can be passed into mod.ts and close ts uses rebase

### PR DESCRIPTION
**Pull request by jsmvaldivia**
- Modified mod.ts mergePullRequest method in order to pass as an option the merge method.
- Modified close command to pass the rebase merge method to mergePullRequest
- Added a missing await in case of an error for the openPullRequestMethod to better log the response

### CODE REVIEW

File: lib/mod.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ export interface MergePullRequestOptions {
   owner: string;
   repo: string;
   number: number;
+  mergeMethod: string;
```
Comment (crqra) [inline lib/mod.ts:25]: Maybe we could add a `MergeMethod` enum just to make it simpler to know the options available?

**Review by crqra — CHANGES_REQUESTED** [review]

File: lib/mod.ts (hunk 1 above)
Comment (jsmvaldivia) [reply lib/mod.ts:25]: That sounds like a sound approach

Comment (jsmvaldivia) [reply lib/mod.ts]: fixed

**Review by crqra — APPROVED** [review]
LGTM! 👍🏼

**Pull request merged**